Build fails with local notmuch

Hi,

I compiled notmuch myself with `--prefix=$HOME` and, while astroid can find it during config/compilation, apparently hypocycloid (?) cannot:

```
Scanning dependencies of target gir_main
[  8%] Building C object CMakeFiles/gir_main.dir/src/plugin/astroid_activatable.c.o
[  9%] Building C object CMakeFiles/gir_main.dir/src/plugin/thread_view_activatable.c.o
[ 10%] Building C object CMakeFiles/gir_main.dir/src/plugin/thread_index_activatable.c.o
[ 11%] Building C object CMakeFiles/gir_main.dir/src/plugin/gir_main.c.o
[ 11%] Linking C executable gir_main
[ 11%] Built target gir_main
Scanning dependencies of target Astroid_0_2_gir_gir
[ 11%] Generating Astroid-0.2.gir
[ 11%] Built target Astroid_0_2_gir_gir
Scanning dependencies of target Astroid_0_2_gir_typelib
[ 12%] Generating Astroid-0.2.typelib
[ 13%] Built target Astroid_0_2_gir_typelib
Scanning dependencies of target gir-typelibs
[ 13%] Built target gir-typelibs
Scanning dependencies of target hypocycloid
[ 14%] Building CXX object CMakeFiles/hypocycloid.dir/src/account_manager.cc.o
In file included from /home/igor/dev/astroid/src/account_manager.cc:11:
/home/igor/dev/astroid/src/db.hh:13:11: fatal error: notmuch.h: No such file or directory
 # include <notmuch.h>
           ^~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/hypocycloid.dir/build.make:71: CMakeFiles/hypocycloid.dir/src/account_manager.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:182: CMakeFiles/hypocycloid.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

I managed to build it with this simple patch:
```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index f87c8ab..51d49c2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -159,6 +159,7 @@ include_directories (
   ${SASS_INCLUDE_DIRS}
   ${Boost_INCLUDE_DIRS}
   ${PROTOBUF_INCLUDE_DIRS}
+  ${Notmuch_INCLUDE_DIRS}
   )
 
 add_compile_options (
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -159,6 +159,7 @@ include_directories (
   ${SASS_INCLUDE_DIRS}
   ${Boost_INCLUDE_DIRS}
   ${PROTOBUF_INCLUDE_DIRS}
+  ${Notmuch_INCLUDE_DIRS}
   )
 
 add_compile_options (

